article/article.md

```diff
@@ -0,0 +1,119 @@
+# How to test a UI Designer page using Cypress
+
+It is not enough to make an UI Designer page, testing it is very important as well. 
+For this reason, our team decided to move towards an e2e framework that will let us have a fast way to set up and test the pages. 
+We decided to use [Cypress](https://www.cypress.io/), an open-source, javascript framework that lets you write Given-When-Then semi-structured tests.
+
+In this article I will go over an UI Designer page that lets you create a user and two tests that were made for the page. 
+You can find all of these in this [github](https://github.com/bonitasoft-labs/bonita-uid-page-cypress-test) repository. 
+The two tests that I will talk about aren’t the only ones that you will find in the repository and if you want to see more examples, you can clone the repository and then check them out by yourself.
+
+### Diving into the page structure
+Let’s start with the page. It has a form container with multiple fields that will ask for different information in order to create a new user. 
+You can also see some messages that will be displayed in case of errors and success, as well as a loading message. 
+There is also a hidden functionality that makes the success or error messages disappear after three seconds if the user makes any action on the page. 
+We usually use this feature in order to mimic a toast and thus not clog the user's screen.
+![page structure](images/page.jpg)
+
+##The test choices
+Without further ado, let’s talk about tests. 
+Cypress tests are made from two distinct parts. 
+The first is about writing the test scenarios in one file and the second is about explaining with code what each line of the scenario will do. 
```

Comment:
I think we should mention that for this part we use cypress-cucumber-preprocessor
Otherwise there is no scenario feature, right ?